src/vivarium_public_health/metrics/utilities.py

```diff
@@ -318,6 +318,43 @@ def get_treatment_counts(pop, config, treatment, doses, event_time, age_bins):
     return dose_counts
 
 
+def get_lived_in_span(pop: pd.DataFrame, t_start: pd.Timestamp, t_end: pd.Timestamp):
+    """Gets a subset of the population that lived in the time span.
+
+    Parameters
+    ----------
+    pop
+        A table representing the population with columns for 'entrance_time',
+        'exit_time' and 'age'.
+    t_start
+        The date and time at the start of the span.
+    t_end
+        The date and time at the end of the span.
+
+    Returns
+    -------
+    lived_in_span
+        A table representing the population who lived some amount of time
+        within the time span with all columns provided in the original
+        table and additional columns 'age_at_start' and 'age_at_end' indicating
+        the age of the individual at the start and end of the time span,
+        respectively.
```

Comment:
if they die in the span I can assume age at end is set to death age ?

Reply:
This function really doesn't know anything about why a simulant is no longer being tracked by the simulation, just that they aren't being tracked.  Should work for people who migrate in/out, exit because they get to old, etc.

Reply:
I shouldn't have used born/died. This was just a language thing but it's good as is, no one probably interprets it like I do :stuck_out_tongue_closed_eyes: